config/7.1.0/securebanking/ig/scripts/groovy/ValidateDebtorAccountInRS.groovy

```diff
@@ -0,0 +1,53 @@
+def fapiInteractionId = request.getHeaders().getFirst("x-fapi-interaction-id");
+if(fapiInteractionId == null) fapiInteractionId = "No x-fapi-interaction-id"
+SCRIPT_NAME = "[ValidateDebtorAccountInRS] (" + fapiInteractionId + ") - "
+
+logger.debug(SCRIPT_NAME + "Running...")
+def method = request.method
+
+
+switch(method.toUpperCase()) {
+
+    case "POST":
+        def debtorAccount = request.entity.getJson().Data.Initiation.DebtorAccount
+        if(debtorAccount !=null){
+            def requestURI = routeArgRsBaseURI + "/backoffice/accounts/search/findByAccountIdentifiers" +
+                    "?name=" + debtorAccount.Name + "&identification=" + debtorAccount.Identification +
+                    "&schemeName=" + debtorAccount.SchemeName
+
+            logger.debug(SCRIPT_NAME + " findByAccountIdentifiers raw request uri: " + requestURI)
+            Request RSRequest = new Request()
```

Comment:
fix variable name to lowerCamel, same for RSResponseStatus

Reply:
done